BinaryObjectScanner.Protection/Macrovision.SafeDisc.cs

```diff
@@ -37,12 +49,32 @@ internal string SafeDiscCheckPortableExecutable(string file, PortableExecutable
             if (sections == null)
                 return null;
 
+            // Found in Redump entry 57986.
+            bool hintNameTableMatch = pex.ImportHintNameTable?.Any(s => s == "LTDLL_Authenticate") ?? false;
+            if (hintNameTableMatch)
+                return "SafeDisc Lite";
+
+            // Found in Redump entry 57986.
+            bool importTableMatch = (pex.ImportTable?.ImportDirectoryTable?.Any(idte => idte.Name == "ltdll.dll") ?? false);
+            if (importTableMatch)
+                return "SafeDisc Lite";
+
+            // Get the .data/DATA section strings, if they exist
+            List<string> strs = pex.GetFirstSectionStrings(".data") ?? pex.GetFirstSectionStrings("DATA");
+            if (strs != null)
+            {
+                // Found in Redump entries 14928, 25579, 32751.
+                if (strs.Any(s => s.Contains("LTDLL_Initialise")))
+                    return "SafeDisc Lite";
+                if (strs.Any(s => s.Contains("LTDLL_Authenticate")))
+                    return "SafeDisc Lite";
+                if (strs.Any(s => s.Contains("LTDLL_Unwrap")))
+                    return "SafeDisc Lite";
+            }
```

Comment:
Suggestion: Move Lite checks to their own partial class. IIRC you mentioned that LT and normal were different enough?

Reply:
There's a very significant overlap with SD1, most of the path checks have overlap.